requirements/test.txt

```diff
@@ -1,54 +1,48 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.10
 # by the following command:
 #
 #    make upgrade
 #
 appdirs==1.4.4
     # via
-    #   -r requirements/base.txt
+    #   -r /Users/qasim/IdeaProjects/openedx/edx-when/requirements/base.txt
```

Comment:
Make upgrade should not have your full local path, you might need a newer version of pip-tools and you should run this on python 3.11